Kickstarter-iOS/Features/LoginTout/Controller/LoginToutViewController.swift

```diff
@@ -573,6 +598,14 @@ extension LoginToutViewController: ASAuthorizationControllerDelegate {
   }
 }
 
+// MARK: SetYourPasswordViewControllerDelegate
+
+extension LoginToutViewController: SetYourPasswordViewControllerDelegate {
+  func postEnvironmentLoggedInNotification() {
```

Comment:
Can we call this `setPasswordCompleteAndLogUserIn`?

Reply:
sure!